mac/macdeploy.sh

```diff
@@ -7,6 +7,7 @@
 
 MACDEPLOYQT="macdeployqt"
 LIBRARY_NAME="libQupZilla.2.dylib"
+PLUGINS="QupZilla.app/Contents/Resources/plugins"
```

Comment:
Btw it will be a dmg... as my u/l speed isn't good with 152MiB folder vs the dmg which is 50.2MiB... just mount the image and then go to /Volumes/QupZilla or install it and then it should be r/w...  uploading now to dune... will send a link in email when it's finished. You'll also need a refresher on `otool -L`.